docs/self-managed/zeebe-deployment/configuration/broker.md

```diff
@@ -0,0 +1,609 @@
+backpressure:
+  enabled: true
+  useWindowed: true
+  algorithm: aimd
+  aimd:
+    requestTimeout: 200ms
+    initialLimit: 100
+    minLimit: 1
+    maxLimit: 1000
+    backoffRatio: 0.9
+  fixed:
+    limit: 20
+  vegas:
+    initialLimit: 20
+    alpha: 3
+    beta: 6
+  gradient:
+    minLimit: 10
+    initialLimit: 20
+    rttTolerance: 2.0
+  gradient2:
+    minLimit: 10
+    initialLimit: 20
+    rttTolerance: 2.0
+    longWindow: 600
+```
+
+### zeebe.broker.exporters
+
+Each exporter should be configured following this template:
+
+| Field     | Description                                                                                                                                                                                                                                                                                                                                            | Example Value |
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------- |
+| exporters | Configure exporters below                                                                                                                                                                                                                                                                                                                              |               |
+| jarPath   | path to the JAR file containing the exporter class. JARs are only loaded once, so you can define two exporters that point to the same JAR, with the same class or a different one, and use args to parametrize its instantiation.                                                                                                                      |               |
+| className | entry point of the exporter, a class which _must_ extend the io.camunda.zeebe.exporter.Exporter interface. A nested table as "args:" will allow you to inject arbitrary arguments into your class through the use of annotations. These setting can also be overridden using the environment variables "`ZEEBE_BROKER_EXPORTERS_`[exporter name]\_..." |               |
+
+#### YAML snippet
+
+```yaml
+exporters
+  jarPath:
+  className:
+```
+
+### zeebe.broker.exporters.debuglog (Debug Log Exporter)
+
+Enable the debug exporter to log the exported records to console.
+
+These setting can also be overridden using the environment variables "`ZEEBE*BROKER_EXPORTERS_DEBUGLOG*`..."
+
+#### YAML snippet
+
+```yaml
+exporters:
+  debuglog:
+    className: io.camunda.zeebe.broker.exporter.debug.DebugLogExporter
+    args:
+      logLevel: debug
+      prettyPrint: false
+```
+
+### zeebe.broker.exporters.debugHttp (Debug HTTP Export)
```

Comment:
:x: Let's remove the debug exporters in general, they aren't really meant for production usage.